voice/src/main/kotlin/VoiceConnectionBuilder.kt

```diff
@@ -142,7 +142,7 @@ public class VoiceConnectionBuilder(
             voiceState.sessionId
         ) to VoiceGatewayConfiguration(
             voiceServer.token,
-            "wss://${voiceServer.endpoint}?v=4"
+            "wss://${voiceServer.endpoint}/?v=4"
```

Comment:
Should this also be changed [here](https://github.com/kordlib/kord/blob/0.8.x/voice/src/main/kotlin/handlers/VoiceUpdateEventHandler.kt#L67)?

Reply:
Yes, good catch.